docs/security/authentication.md

```diff
@@ -115,9 +116,6 @@ This instruction based on Ubuntu 14.04 LTS but may work with other OS with few c
     ```
     Then check HTTP Basic Authentication works in browser. If you can see regular basic auth popup and then able to login with credential you entered into `.htpasswd` you are good to go.
 
-    <img src="/assets/themes/zeppelin/img/screenshots/authentication-basic-auth-nginx-request.png" />
-    <img src="/assets/themes/zeppelin/img/screenshots/authentication-basic-auth-nginx-https.png" />
```

Comment:
Would you also remove "docs/assets/themes/zeppelin/img/screenshots/authentication-basic-auth-nginx-request.png" and  "docs/assets/themes/zeppelin/img/screenshots/authentication-basic-auth-nginx-https.png"  ?